src/sentry/api/endpoints/organization_events.py

```diff
@@ -68,7 +67,7 @@ def rate_limit_events(request: Request, organization_slug=None, *args, **kwargs)
     try:
         organization = Organization.objects.get_from_cache(slug=organization_slug)
     except Organization.DoesNotExist:
-        raise ResourceDoesNotExist
+        return DEFAULT_RATE_LIMIT_CONFIG
```

Comment:
Would the IP rate limit fit better here?

Reply:
updated to use 50/s and 50 concurrent by default based on slack convo